can/can.proto

```diff
@@ -7,7 +7,7 @@ option cc_enable_arenas = true;
 message SystemTime {
   optional uint32 SID = 1;
   optional uint32 date = 4; // 2 bytes, days since epoch
-  optional uint32 time = 5; // 4 bytes, seconds since midnight
+  optional uint32 time = 5; // 4 bytes, 1/10000 seconds since midnight
```

Comment:
Do the units actually change, or was it just improperly commented at the beginning?

Reply:
Improperly commented at the start.